reactapp/src/Assignment5/Question15/withLogger.jsx

```diff
@@ -0,0 +1,20 @@
+
+import React, { useEffect } from "react";
+
+const withLogger = (WrappedComponent) => {
+  const WithLogger = (props) => {
+    useEffect(() => {
+      console.log(`Component ${WrappedComponent.name} mounted.`);
+      console.log(`Component ${WrappedComponent.name} updated.`)
```

Comment:
use a useRef variable to track mounting, use it show mount only once, and show updated in subsequent rerenders